src/Files.App/Helpers/Win32/Win32Helper.Process.cs

```diff
@@ -13,6 +13,7 @@ namespace Files.App.Helpers
 	public static partial class Win32Helper
 	{
 		private static bool? isRunningOnArm = null;
+		[Obsolete]
```

Comment:
Is there a replacement?

Reply:
Yes WindowsStorablesHelpers.IsOnArmProcessor @ WindowsStorablesHelpers.Process.cs which is newly created in this PR.

Reply:
Yes